spec/features/user_creates_a_new_review_spec.rb

```diff
@@ -0,0 +1,117 @@
+require "rails_helper"
+
+feature 'user can add a new review for a theme song', %(
+As a user I can review television program theme song
+So that I can submit a rating of how much
+I enjoy it, and optionally a comment on the song!
+) do
+
+# Acceptance Criteria
+#   [ ] Users must input a rating from 1 to 5
+#   [ ] Users can optionally leave a comment, less than 200 characters
+#   [ ] When the user submits the review, the user review is displayed on the show page
+#   [ ] A flash message to indicate success
+#   [ ] A flash message to indicate failure
+
+  scenario 'user adds a review' do
+    user = FactoryGirl.create(:user)
+    program_1 = FactoryGirl.create(:program)
+
+    sign_in(user)
+    expect(page).to have_content('Signed in successfully')
+
+    review = FactoryGirl.create(
+    :review,
```

Comment:
Align the parameters of a method call if they span more than one line.<br>Indent the first parameter one step more than the previous line.